wagtail/wagtailimages/models.py

```diff
@@ -211,7 +212,7 @@ def get_suggested_focal_point(self):
         return Rect.from_point(x, y, width, height)
 
     def get_rendition(self, filter):
-        if not hasattr(filter, 'run'):
+        if isinstance(filter, string_types):
             # assume we've been passed a filter spec string, rather than a Filter object
             # TODO: keep an in-memory cache of filters, to avoid a db lookup
```

Comment:
Might be worth killing this comment while you're at it. We plan on making ``Filter`` into a standard python class instead